lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/YarnSubmissionHelper.java

```diff
@@ -141,7 +141,7 @@ public YarnSubmissionHelper setApplicationName(final String applicationName) {
    * @return
    */
   public YarnSubmissionHelper setDriverMemory(final int megabytes) {
-    applicationSubmissionContext.setResource(Resource.newInstance(getMemory(megabytes), 1));
+    applicationSubmissionContext.setResource(Resource.newInstance(getMemory(megabytes), 6));
```

Comment:
What does this change from `1` to `6` represent? Doesn't this change the number of cores for the Driver? This is dangerous, as we might run on a cluster where no container with `6` cores can be allocated ever.

Reply:
I created REEF-1896 to make it configurable from client. This is for Yarn submission helper rather than local. So multiple cores usually should be available. Before we complete REEF-1896, default 1 is too low for our client.

Reply:
I understand. But I can get a HDI cluster with 4 cores per machine, and then none of the Drivers would work. We really need to make this configurable, and set the default to `1` for maximum compatibility.